packages/ui/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@ballerine/ui",
+  "private": false,
+  "version": "0.0.1",
+  "type": "module",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./lib/index.js",
+      "types": "./lib/index.d.ts"
+    },
+    "./components/*": "./lib/components/*",
+    "./lib/style.css": "./lib/style.css"
+  },
+  "scripts": {
+    "dev": "vite build --watch",
+    "build": "tsc && vite build",
+    "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview",
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build",
+    "test": "vitest run"
+  },
+  "dependencies": {
+    "@radix-ui/react-dialog": "^1.0.2",
+    "@radix-ui/react-dropdown-menu": "^2.0.5",
+    "@radix-ui/react-label": "^2.0.1",
+    "@radix-ui/react-scroll-area": "^1.0.2",
+    "@radix-ui/react-slot": "^1.0.1",
+    "@tanstack/react-table": "^8.9.2",
+    "class-variance-authority": "^0.6.0",
+    "clsx": "^1.2.1",
+    "dayjs": "^1.11.6",
+    "lodash": "^4.17.21",
+    "lucide-react": "^0.144.0",
+    "react-custom-scrollbars": "^4.2.1",
```

Comment:
Remove `react-custom-scrollbars` if it's no longer in use. :slightly_smiling_face: